examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py

```diff
@@ -208,8 +227,12 @@ def main(core_context):
         # NEW: Save checkpoint.
         checkpoint_metadata_dict = {"steps_completed": steps_completed}
 
+        # NEW: Here we are saving multiple files to our checkpoint directory, a model weights file as well as a file that includes other
+        # information about the checkpoint, in this case, epochs completed and the trial ID.
```

Comment:
Break this comment into multiple sentences, or bullet points, or something.  It is not very understandable.

Also I recommend the phrases "model state" and "training loop state" or something to that effect in order to illustrate the difference, but up to you.